go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/docker/docker v27.5.1+incompatible
 	github.com/edsrzf/mmap-go v1.2.0
 	github.com/foxboron/go-uefi 69fb7dba244f
+  github.com/gofrs/flock v0.8.1
```

Comment:
something is wrong with indentation or Github doesn't style it correctly?

Reply:
seems a real issue, somehow I screwed the indentation